yt/frontends/enzo_p/tests/test_misc.py

```diff
@@ -51,6 +51,11 @@ def test_get_block_info():
         assert left == float(n) / max_n
         assert right == float(n+1) / max_n
 
+    level, left, right = get_block_info("B__")
+    assert level == 0
+    assert (left == np.zeros(3)).all()
```

Comment:
You don't have to change this, but isn't this the same as `(left == 0.0).all()` and likewise for `1.0` below?

Reply:
Yes, this is the same. I addressed it in the new commit.